modules/i3s/src/index.js

```diff
@@ -1,5 +1,3 @@
-export {I3SNodeGeometryLoader} from './i3s-node-geometry-loader';
```

Comment:
What happened to these?

Reply:
the `I3SNodeGeometryLoader` is not implemented yet, so just cut it for now.